docs/modules/device-lab-management/pages/ios-devices/add-ios-device.adoc

```diff
@@ -0,0 +1,318 @@
+
+If the Mac mini host does not have Internet connection, follow the section xref:#preload-ddi-air-gapped[preload DDI for air-gapped Mac mini] before continuing with this section.
+
+====
+
+Follow the appropriate steps based on whether you are using Standard or Lightning mode
+
+[tabs]
+====
+
+Standard Mode::
++
+--
+
+Open Xcode on the Mac mini host, then navigate to **Window → Devices and Simulators**. Do this before continuing to the next step.
+
+The Trust *this computer* prompt on the device screen appears, tap **Trust**.
+
+image::device-lab-management:device-lab-management-ios-add-ios-standard-lightning-trust.PNG[width=300,alt="Standard Mode. Trust This Computer popup, clicking Trust"]
+
+The Trust *this computer* prompts will reappear, tap **Trust** again. This time there should be no more **Trust** prompts.
+
+--
+
+Lightning mode::
++
+--
+
+Open Xcode on the Mac mini host, then navigate to **Window → Devices and Simulators**. Do this before continuing to the next step.
+
+Make sure you connect the device **to the Mac mini host** first.
+
+The Trust *this computer* prompt on the device screen appears, tap **Trust**.
+
+image::device-lab-management:device-lab-management-ios-add-ios-standard-lightning-trust.PNG[width=300,alt="Lightning Mode. Trust This Computer popup, clicking Trust"]
+
+In the Mac mini host’s screen, under the **Devices** tab of the **Devices and Simulators** screen, the iOS 17 devices should show up with a yellow warning message like the one below:
+
+image::device-lab-management:device-lab-management-ios-add-ios-device-and-simulator.PNG[width=600,alt="iOS 17 in Devices and Simulators"]
+
+Unplug the device from the Mac mini host and plug it into the GEM.
+
+The *Trust this computer* prompts will reappear, tap **Trust** again. This time there should be no more **Trust** prompts.
+
+In Xcode’s Devices and Simulators, the iOS 17 devices now display with a globe icon next to it like below:
+
+image::device-lab-management:device-lab-management-ios-add-ios-xcode-device-and-simulator.PNG[width=300,alt="Devices, iphone is connected notification"]
+
+--
+
+====
+
+Wait until the device screen changes to the below before continuing. There will also be an *automation running* overlay above the device screen (not shown in screenshot).
+
+image::device-lab-management:device-lab-management-add-android-screen-changes-to-blue.PNG[width=300, alt="device screen changes and shows Kobiton name and logo"]
+
+[#preload-ddi-air-gapped]
+=== Preload DDI for air-gapped Mac mini hosts
+
+TIP: This section is only required for Mac mini hosts with no Internet access with iOS 17 and above devices.
+
+Acquire any MacOS machine with Internet access. This will be referred to as the Internet Mac.
+
+Ensure Xcode is installed on the Internet Mac. Make sure the Xcode version is compatible with the iOS 17 device. Note: Kobiton software such as deviceConnect and deviceShare do not need to be installed on the Internet Mac.

```

Comment:
Note: Kobiton software, such as deviceConnect and deviceShare, does not need to be installed on the Internet Mac.

Reply:
updated